QUEST.tsv

```diff
@@ -2707,30 +2707,30 @@ QUEST_20151016_002714	Only you...{nl}Only the Revelator can save this world.{nl}
-QUEST_20151016_002718	{memo X}
-QUEST_20151016_002719	
-QUEST_20151016_002720	{memo X}
-QUEST_20151016_002721	{memo X}
-QUEST_20151016_002722	{memo X}
-QUEST_20151016_002723	{memo X}
-QUEST_20151016_002724	{memo X}
-QUEST_20151016_002725	{memo X}
-QUEST_20151016_002726	{memo X}
-QUEST_20151016_002727	{memo X}
-QUEST_20151016_002728	{memo X}
-QUEST_20151016_002729	{memo X}
-QUEST_20151016_002730	{memo X}
-QUEST_20151016_002731	{memo X}
-QUEST_20151016_002732	{memo X}
+QUEST_20151016_002718	{memo X}This vicinity is Kedoran Merchant Alliance's excavation site.
+QUEST_20151016_002719	Received Kedoran Merchant Alliance's... benefit.{nl}Somehow I'm full of energy.
+QUEST_20151016_002720	{memo X}Kedoran Merchant Alliance's good is near the upper right corner.
+QUEST_20151016_002721	{memo X}Kedoran Merchant Alliance's good is near upper end.
+QUEST_20151016_002722	{memo X}Kedoran Merchant Alliance's good is near the upper left corner.
+QUEST_20151016_002723	{memo X}Kedoran Merchant Alliance's good is a bit far away in the upper right corner.
+QUEST_20151016_002724	{memo X}Kedoran Merchant Alliance's good is a bit far away in the upper end.
+QUEST_20151016_002725	{memo X}Kedoran Merchant Alliance's good is a bit far away in the upper left corner.
+QUEST_20151016_002726	{memo X}Kedoran Merchant Alliance's good is a far away in the upper right corner.
+QUEST_20151016_002727	{memo X}Kedoran Merchant Alliance's good is a far away in the upper end.
+QUEST_20151016_002728	{memo X}Kedoran Merchant Alliance's good is a far away in the upper left corner.
+QUEST_20151016_002729	{memo X}They say what is the use in the world with disappeared goddesses, but we believe Goddesses will one day return and save us.{nl}Is it wrong to pray so that the day will come sooner?{nl}These monuments were built with our devotion from our praying hearts.{nl}Until some strange person came and destroyed the monuments, it symbolized our prayers.
+QUEST_20151016_002730	{memo X}If it's a normal monument, we prayed to release such an aura.[nl]It's not some corruption!{nl}Working under someone who neither understand or can hear, did the person who destoryed our monuments saw it as a corruption?
+QUEST_20151016_002731	{memo X}Do you finally understand?{nl}What your utter stupidity has caused?{nl}If you understand, start trying to think of a way to revert this.{nl}This can't simply be ignored.
+QUEST_20151016_002732	{memo X}Below, I believe there's someone destroying Nestos's Monuments by disguising as Anastos.{nl}Can you go and investigate?{nl}It might be dangerous, take this flare.{nl}Be careful.
 QUEST_20151016_002733	$Kedoran Merchant Alliance Notice
 QUEST_20151016_002734	$Recruiting new members for the Kedoran Merchant Alliance!
-QUEST_20151016_002735	
-QUEST_20151016_002736	
-QUEST_20151016_002737	
-QUEST_20151016_002738	
-QUEST_20151016_002739	
-QUEST_20151016_002740	
-QUEST_20151016_002741	
+QUEST_20151016_002735	Nestos's Member
```

Comment:
I found that they used "Nestos" in etc.tsv so I stuck with that translation. Nestopa works better though.